### PR TITLE
Misc updates

### PR DESCRIPTION
**Pull request by jmhauck**

### CODE REVIEW

File: src/components/solution-contents/solution-contents.tsx [hunk 1]
```diff
@@ -139,14 +139,21 @@ export class SolutionContents {
   //--------------------------------------------------------------------------
 
   /**
-   * Publishes the `solutionItemSelected` event containing `itemId`, the id of the selected item.
+   * Publishes the `solutionItemSelected` event containing `solutionItem` of the selected item.
+   * 
+   * Also toggles the expanded state of the tree item.
    *
    * @param solutionItem the
```
Comment (MikeTschudi) [inline src/components/solution-contents/solution-contents.tsx:146]: `solutionItem` the...?

File: src/components/solution-variables/solution-variables.tsx [hunk 2]
```diff
@@ -142,4 +142,18 @@ export class SolutionVariables {
...
+   * Toggle the tree item that was clicked
+   *
+   * @param evt the clicks mouse event
+   */
+  private _toggleExpand(
+    evt: any = undefined
+  ): void {
+    const treeItem = evt?.target?.closest("calcite-tree-item");
+    if (treeItem) {
+      treeItem.expanded = !treeItem?.expanded;
```
Comment (MikeTschudi) [inline src/components/solution-variables/solution-variables.tsx:156]: The "?" seems redundant here because you have the check on line 155.

**Review by MikeTschudi — APPROVED** [review]

File: src/components/solution-variables/solution-variables.tsx (hunk 2 above)
Comment (jmhauck) [reply src/components/solution-variables/solution-variables.tsx:156]: yup...not necessary...thanks

File: src/components/solution-contents/solution-contents.tsx (hunk 1 above)
Comment (jmhauck) [reply src/components/solution-contents/solution-contents.tsx:146]: will update

**Pull request merged**